docs/preview/features/openapi/security-definitions.md

```diff
@@ -5,8 +5,12 @@ layout: default
 
 # Adding OAuth security definition to API operations
 
-When an API is secured via OAuth, it is helpful if the Open API documentation makes this clear via a security scheme and the API operations that require authorization automatically inform the consumer that it is possible that a 401 Unauthorized or 403 Forbidden response is returned.
-The `OAuthAuthorizeOperationFilter` that is part of this package exposes this functionality.
+When an API is secured via OAuth, [Shared Access Key authentication](../../features/security/auth/shared-access-key), [Certificate authentication](../../features/security/auth/certificate), it is helpful if the Open API documentation makes this clear via a security scheme and the API operations that require authorization automatically inform the consumer that it is possible that a 401 Unauthorized or 403 Forbidden response is returned.
+
+These `IOperationFilter`'s that are part of this package exposes this functionality:
```

Comment:
This is a major nit, but can we alphabetically order them?

Reply:
Yeah, sure. Done.